build.sbt

```diff
@@ -1202,6 +1202,7 @@ lazy val `sbt-plugin` = (project in file("dev") / "sbt-plugin")
       val () = (publishLocal in `client-scaladsl`).value
       val () = (publishLocal in `cluster-core`).value
       val () = (publishLocal in `cluster-javadsl`).value
+      val () = (publishLocal in `cluster-scaladsl`).value
```

Comment:
Scripted tests failed because `cluster-scaladsl` was not being locally published before the tests are run. This should do the trick now.